service/src/main/java/bio/terra/catalog/api/DatasetApiController.java

```diff
@@ -88,4 +88,10 @@ public ResponseEntity<DatasetPreviewTable> getDatasetPreviewTable(UUID id, Strin
         .cacheControl(CacheControl.noStore())
         .body(datasetService.getDatasetPreview(getUser(), new DatasetId(id), tableName));
   }
+
+  @Override
+  public ResponseEntity<Void> exportDataset(UUID datasetId, UUID workspaceId) {
+    datasetService.exportDataset(getUser(), new DatasetId(datasetId), workspaceId.toString());
```

Comment:
Is there a reason to convert the workspace ID into a string at this layer? Would it be incorrect to make the `DatasetService` API use UUIDs? I think it would be nice to use a more specific type in our code if possible.